proposals/lip-0045.md

```diff
@@ -599,7 +630,8 @@ chainIDSchema = {
     "required": ["ID"],
     "properties": {
         "ID": {
```

Comment:
I think `chainID` would be more clear.

Reply:
Note that below you also use a different property name

> The store value is an object with `chainID=MAINCHAIN_ID`, serialized using the JSON schema `chainIDSchema`.

Here I would also recommend to write `chainID == MAINCHAIN_ID`

Reply:
fixed